contents/docs/tutorials/actions.md

```diff
@@ -0,0 +1,196 @@
+
+The relationship between events and actions can be both one-to-one or many-to-one. What this means is that you can create an action from one event only, such as a "Pageview" action for a `$pageview` event, as well as an action targeting multiple events, like a "Sign Up" action targeting the events `clicked button with text "Sign Up"` and `clicked button with text "Sign Up Now"`.
+
+Not only that, you can also create actions based on the URL where an event happened. This could lead to actions such as "Clicked Sign Up on About Page".
+
+There are two main reasons for creating actions:
+
+**1. Tagging events that you especially care about**
+
+Filter out the noise from all your events to make it easier for you to execute your analytics processes.
+
+**2. Combining multiple events into one action**
+
+Let's say you have multiple sign up buttons spread across your website. For some types of analysis it could be useful to know which specific button was clicked, however, it might also be the case that you just care that the user signed up, irrespective of what they clicked to do so.
+
+Actions fit this use-case perfectly, since events are combined using `OR` operations. That means that if an action is composed of multiple events, it will be registered if _any of the events composing it_ are triggered. 
+
+In other words, using the example from above, if you have an action called "Sign Up" that targets the clicks on all your sign up buttons, if your user clicks any of the buttons, the action will be registered.
+
+> **Note:** Actions also work retroactively, meaning that they also tag events that happened in the past, not only events that happened after the action was created.
+
+### Creating Actions
+
+There are many ways to create actions in PostHog. These are:
+
+**1. Through our toolbar (recommended for actions based on frontend elements)**
+
+To learn how to create actions using our toolbar, you can visit our [dedicated tutorial](/docs/tutorials/toolbar/#creating-actions).
+
+**2. From the events table**
+
+To create actions from the events table, just click on the plus sign on the left of any event to reveal a button allowing you to create an action from that specific event. You will also be able to customize the action further in the following steps (such as add another event). 
+
+![Action from event image](../../images/tutorials/actions/action-from-event.png)
+<small class='centered'>_Creating actions from the events table_</small>
+
+
+**3. From the 'Actions' page**
+
+Lastly, you can of course create actions from the 'Actions' page under 'Events':
+
+![Actions page image](../../images/features/actions/actions-page.png)
+
+
+**Action Options**
+
+Visit our [dedicated actions page](/docs/features/actions/) for a comprehensive explanation of all the options available when creating an action.
+
+### Upgrading Your Analytics Game with Custom Events
```

Comment:
Could we word this from a use case perspective? "Upgrading your analytics game" is vague.